dxp/exporting.md

```diff
@@ -0,0 +1,47 @@
+# Exporting your JavaScript
+
+This document describes how you should organize you npm packages to export JavaScript functionality
+
+## Introduction
+
+Although you usually consume packages via [npm](https://npmjs.com), you are very likely to create packages yourself: for example if you want to expose useful functionality and be able to reuse it elsewhere.
```

Comment:
> I think we should be careful how we use the word "package" here though, because we are specifically talking about fake "packages" that correspond to OSGi modules in DXP.

Yes that's a very good point. I'll make that clear by explaining that these are "fake" packages and something that we only do in the context of DXP.